packages/postcss-convert-values/src/__tests__/index.js

```diff
@@ -67,6 +67,11 @@ test(
   processCSS('h1{width:100.00%}', 'h1{width:100%}')
 );
 
+test(
+  'should preserve opacities defined as percentages',
+  passthroughCSS('h1{opacity:100%}')
+);
+
```

Comment:
Can you add some more.

may be like these
⬇️ 

```js
test(
  'should convert the opacity to 1px',
  processCSS('h1{opacity : 80px}', 'h1{opacity: 1px}')
);
test(
  'should convert the opacity from 80% -> 80%',
  processCSS('h1{opacity :80%}', 'h1{opacity :80%}')
);
```

Also check this one 
⬇️ 
```js
test(
  'should convert the opacity without any unit',
  passthroughCSS('h1{opacity :80}')    // expected is 80
);

```

Reply:
Regarding your tests:
```
test(
  'should convert the opacity to 1px',
  processCSS('h1{opacity : 80px}', 'h1{opacity: 1px}')
);
```
'px' is not a valid unit, and I opted not to check for invalid units as that's not the tool's purpose (minification vs linting)

---
```
test(
  'should convert the opacity from 80% -> 80%',
  processCSS('h1{opacity :80%}', 'h1{opacity :80%}')
);
```
If I'm not mistaken, `processCSS('h1{opacity :80%}', 'h1{opacity :80%}')` would be the same test as `passthroughCSS('h1{opacity:80%}')` or the test I created, `passthroughCSS('h1{opacity:100%}')`. Please correct me if I'm wrong about that.

---
```
test(
  'should convert the opacity without any unit',
  passthroughCSS('h1{opacity :80}')    // expected is 80
);
```
Any number above '1.0' is invalid and would be converted to '1'. There's a check for this at line 281:
```
test(
  'should clamp opacity to 1 maximum',
  processCSS(
    'h1{opacity:150;opacity:15;opacity:1.5}',
    'h1{opacity:1;opacity:1;opacity:1}'
  )
);
```
---
The only other tests I can think that aren't already tested for are for validating unit type, but that would be a linter's job.

Reply:
> Regarding your tests:
> ```js
> test(
>  'should convert the opacity to 1px',
>  processCSS('h1{opacity : 80px}', 'h1{opacity: 1px}')
> );
> ```
> 
> 'px' is not a valid unit, and I opted not to check for invalid units as that's not the tool's purpose (minification vs linting)

Invalid CSS should lean to either no minification or invalid minification. This is an example of invalid minification. 
You can keep it or omit it. either looks good. 
_but I would check the behavior by keeping it for future use_

> If I'm not mistaken, processCSS('h1{opacity :80%}', 'h1{opacity :80%}') would be the same test as passthroughCSS('h1{opacity:80%}') or the test I created, passthroughCSS('h1{opacity:100%}'). Please correct me if I'm wrong about that.

Yea, but I prefer using `processCSS` as its more visible. again your choice to use either of them. but they both are fine 👍 

> Any number above '1.0' is invalid and would be converted to '1'. There's a check for this at line 281:

if its done,  leave it then 👍 

> The only other tests I can think that aren't already tested for are for validating unit type, but that would be a linter's job.

Feel free to choose whether to test it or not. @evilebottnawi what you think?

Reply:
I prefer to use `passthroughCSS` when values was not changed, I think in future we refactor tests on snapshots, because it is more easy to read